data/styles/vehicle/include/priorities_4_overlays.prio.txt

```diff
@@ -516,6 +516,7 @@ amenity-charging_station-bicycle                    # icon z16- (also has captio
 === 850
 
 historic-boundary_stone                             # icon z16- (also has caption(optional) z16-)
+boundary=marker                                     # icon z16- (also has caption(optional) z16-)
```

Comment:
it should be removed from the vehicle style

Reply:
Done, thanks